elastic/system/ksp-allow-elastic-access-file-sys-proc.yaml

```diff
@@ -0,0 +1,31 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name:ksp-allow-elastic-access-file-sys-proc
```

Comment:
@AbhinavCSY Space needed after the name.

Reply:
added @salman-accuknox